genderbias/publications/__init__.py

```diff
@@ -3,7 +3,7 @@
 """
 from typing import List, Dict
 
-import re
+import re, math
```

Comment:
leaving it now, but in general the convention is for each package import to be on its own line. (We can hit this in the future!)

Reply:
Sorry about that, I'll remember in the future! And it looks like #72 gets us headed towards automated linting, which would have prevented this.

Thanks for merging anyway this one time. It was awesome to be able to count #62 as done. We had a remarkably solid first sprint.

Reply:
Yes!! Huge huge success! :D